sync/downstream.py

```diff
@@ -257,24 +261,14 @@ def tried_to_rebase(self, value: bool) -> None:
         self.data["tried_to_rebase"] = value
 
     @mut()
-    def try_rebase(self) -> DownstreamAction | None:
-        if self.tried_to_rebase:
-            return DownstreamAction.manual_fix
-        else:
-            try:
-                logger.info("Rebasing onto %s" % self.gecko_landing_branch())
-                self.tried_to_rebase = True
-
-                commit_hash_before_rebase = self.gecko_commits.base.sha1
-                self.gecko_rebase(self.gecko_landing_branch(), abort_on_fail=True)
-                commit_hash_after_rebase = self.gecko_commits.base.sha1
+    def try_rebase(self) -> None:
+        try:
+            logger.info("Rebasing onto %s" % self.gecko_landing_branch())
+            self.tried_to_rebase = True
 
-                if commit_hash_before_rebase == commit_hash_after_rebase:
-                    return DownstreamAction.manual_fix
-                else:
-                    return None
-            except AbortError:
-                return DownstreamAction.manual_fix
+            self.gecko_rebase(self.gecko_landing_branch(), abort_on_fail=True)
+        except AbortError:
+            pass
```

Comment:
I think we should allow the error to propagate in this case.

Reply:
alright, removed try-catch